README.md

```diff
@@ -78,17 +78,20 @@ a) `REACT_APP_LAUNCH_URL=http://example.com PORT=6000 npm start` or b) by specif
+| REACT_APP_DEFAULT_CLIENT_ID       | `app-login`                                                                           | When logging into the EHR, the app will use the provided client id in the request for an authorization code. This variable should be changed if this app is registered under a different client name in the EHR. You can also use the `/register` page to manage client ids for multiple EHR's. |
+| REACT_APP_DEFAULT_ISS             | `http://localhost:8080/test-ehr/r4`                                                   | This is the base url of the EHR that the app will attempt to authenticate against when launched standalone. This URL is not used when the app is launched from an EHR.                                                                                                                          |
+| REACT_APP_DEVELOPER_MODE          | `true`                                                                                | When set to 'true', enables developer functions like allowing forms to be submitted without actually filling out all required fields.                                                                                                                                                           |
+| REACT_APP_ETASU_STATUS_ENABLED    | `true`                                                                                | When set to 'true', shows the ETASU status on the main page. This allows the user to see progress towards completion of the REMS requirements.                                                                                                                                                  |
+| REACT_APP_PHARMACY_STATUS_ENABLED | `true`                                                                                | When set to 'true', shows the pharmacy status on the main page. This allows the user to track progress towards dispensing the medication from the pharmacy.                                                                                                                                     |
+| REACT_APP_REMS_ADMIN_SERVER_BASE  | `http://localhost:8090`                                                               | The base url of the REMS admin server, which handles the ETASU and questionnaires. Should be changed to match the base url of the REMS admin you wish to submit information to.                                                                                                                 |
+| INTERMEDIARY_ETASU_MET            | `http://localhost:3003/4_0_0/GuidanceResponse/$rems-etasu`                            | This is the REMS intermediary url used to make ETASU checks, which the former forwards to a REMS admin.                                                                                                                                                                                         |
```

Comment:
I think this should be the FHIR url instead of directly to the ETASU check. It'll make it more flexible for the future and allow for adding in the /GuidanceResponse/$rems-etasu in a single file.